hack/builder/Makefile

```diff
@@ -8,8 +8,12 @@ BUILD_SHA ?= $$(git rev-parse --short HEAD)
 BUILD_EDITION=tce-standalone
```

Comment:
could we leave builder unchanged? effectively shipping standalone exactly the way we did last release?

Reply:
We would need to roll back a big chunk of this PR then: https://github.com/vmware-tanzu/community-edition/commit/dccd271d08c092c69415d8e2dbcfa8cafe017c69#diff-178629e8a6889c3f38c18d928d26c514ca5486f801b8129b5821dfe3b6794b11

TLDR: we wouldn't be able to use the new LD Flag `buildinfo.Version` present in `tanzu-framework` v0.10.0 and would have to use `cli.BuildVersion` from v0.2.1. So there _is_ an unfortunate coupling between versions pulled in by the plugins and what we build with due to this LD flag.

Reply:
giving this a try now - rolling the builder back and will add as a new commit

Reply:
that didn't work. Rolled everything back to the older version of builder we use and still getting:

```
❯ tanzu standalone-cluster create hello-world -i docker

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
Warning - Standalone clusters will be deprecated in a future release of Tanzu Community Edition
                                   Use at your own Risk
           Checkout the proposal for the standalone cluster replacement:
           https://github.com/vmware-tanzu/community-edition/issues/2266
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

Downloading TKG compatibility file from 'projects.registry.vmware.com/tce/compatability/v0.10.0/tkg-compatibility'
Error: unable to create Tanzu Standalone Cluster client
Cause: unable to ensure prerequisites: unable to ensure tkg BOM file: failed to download default bom files from the registry: unable to get the default BOM file ImagePath and Image Tag from the TKG Compatibility file: unable to find the supported TKG BOM version for the management plugin version "v0.2.1" in the TKG Compatibility file "/home/jmcb/.config/tanzu/tkg/compatibility/tkg-compatibility.yaml"
Error: exit status 1

✖  exit status 1

```

since we are passing the `TKG_DEFAULT_IMAGE_REPOSITORY` into the builder